sdv/constraints/tabular.py

```diff
@@ -1028,11 +1029,18 @@ def _sample_constraint_columns(self, table_data):
                         proposed_table_data[column] = 1.0
 
             else:
+                should_transform = True
+                conditions = sub_table_data[condition_columns]
+                transformed_conditions = self._hyper_transformer.transform(conditions)
                 proposed_table_data = self._columns_model.sample(
                     num_rows=sub_table_data[condition_columns].shape[0],
```

Comment:
Isn't this just `num_rows=len(sub_table_data)`?

Reply:
Hmm I think so, Idk why this is like that